foreman/foreman.spec

```diff
@@ -1004,19 +1012,22 @@ exit 0
 %{foreman_rake} db:seed >> %{_localstatedir}/log/%{name}/db_seed.log 2>&1 || :
 %{foreman_rake} apipie:cache:index >> %{_localstatedir}/log/%{name}/apipie_cache.log 2>&1 || :
 (/sbin/service foreman status && /sbin/service foreman restart) >/dev/null 2>&1
+(/sbin/service %{executor_service_name} status && /sbin/service %{executor_service_name} restart) >/dev/null 2>&1
```

Comment:
Also, are these service start and restarts the smart way to go? I feel like our application is so complex, and so much is baked into our installer that doing these operations outside of it can have unintended consequences.